src/appconfig.c

```diff
@@ -2025,6 +2025,7 @@ ni_config_parse_system_updater(ni_extension_t **list, xml_node_t *node)
 {
 	ni_extension_t *ex, *old;
 	const char *name;
+    const char *format;
```

Comment:
```suggestion
	const char *format;
```

Reply:
Sorry, forgot to change that one to tabs. Fixed with new commit.